NEWS.md

```diff
@@ -220,6 +220,9 @@ This section lists changes that do not have deprecation warnings.
     `airybiprimex`, `airyaix`, `airybix`, `airyaiprimex`, `airybiprimex`)
     ([#18050]).
 
+  * When a macro is called in the module in which that macro is defined, global variables
+    in the macro are now correctly resolved in the macro definition environment ([#15850]).
```

Comment:
Good call! How does the present iteration look? Roughly what you had in mind, or had you something else / additional in mind? Thanks!